packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -116,80 +116,54 @@ def test_octarine_srv(cluster, timeout=30):
     cluster.deploy_marathon_app(app_definition)
 
 
-# By default telemetry-net sends the metrics about once a minute
-# Therefore, we wait up till 2 minutes and a bit before we give up
```

Comment:
nit: This comment about why the timeout is 125 seems like it's still relevant / useful here